File: LLM-Translations/GPT-4/codenet/C++/C/s615507008.c
#include <stdio.h>

void solve(long long N, long long T[], long long M, long long P[], long long X[]){
    for(int i = 0; i < M; i++) {
        long long ans = 0;
        for(int j = 0; j < N; j++) {
            if (j == P[i] - 1) ans += X[i];
            else ans += T[j];
        }
        printf("%lld\n", ans);
    }
}

int main(){
    long long N;
    scanf("%lld", &N);
    long long T[N];
    for(int i = 0; i < N; i++) scanf("%lld", &T[i]);
    long long M;
    scanf("%lld", &M);
    long long P[M], X[M];
    for(int i = 0; i < M; i++) scanf("%lld %lld", &P[i], &X[i]);
    solve(N, T, M, P, X);
    return 0;
}

// 

File: LLM-Translations/GPT-4/codenet/C++/Java/s615507008.java
import java.util.Scanner;
import java.util.stream.Stream;

public class s615507008 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        long[] T = new long[N];
        for (int i = 0; i < N; i++) {
            T[i] = sc.nextLong();
        }
        int M = sc.nextInt();
        long[] P = new long[M];
        long[] X = new long[M];
        for (int i = 0; i < M; i++) {
            P[i] = sc.nextLong();
            X[i] = sc.nextLong();
        }
        solve(N, T, M, P, X);
        sc.close();
    }

    public static void solve(int N, long[] T, int M, long[] P, long[] X) {
        for (int i = 0; i < M; i++) {
            long ans = 0;
            for (int j = 0; j < N; j++) {
                if (j == P[i] - 1) {
                    ans += X[i];
                } else {
                    ans += T[j];
                }
            }
            System.out.println(ans);
        }
    }
}

// 
